engine/src/main/java/org/terasology/rendering/dag/nodes/ApplyDeferredLightingNode.java

```diff
@@ -45,46 +46,53 @@
     @In
     private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
 
+    private FBO sceneOpaqueFbo;
+    private FBO sceneOpaquePingPongFbo;
+    private FBO refractiveReflectiveFbo;
+
     /**
      * Initializes an instance of this node.
      *
      * This method -must- be called once for this node to be fully operational.
      */
     @Override
     public void initialise() {
+        sceneOpaqueFbo = displayResolutionDependentFBOs.get(SCENE_OPAQUE);
+        sceneOpaquePingPongFbo = displayResolutionDependentFBOs.get(SCENE_OPAQUE_PING_PONG);
+        refractiveReflectiveFbo = displayResolutionDependentFBOs.get(REFRACTIVE_REFLECTIVE);
 
-        addDesiredStateChange(new SetViewportToSizeOf(WRITE_ONLY_GBUFFER));
+        addDesiredStateChange(new SetViewportToSizeOf(SCENE_OPAQUE_PING_PONG, displayResolutionDependentFBOs));
         addDesiredStateChange(new EnableMaterial(DEFERRED_LIGHTING_MATERIAL.toString()));
 
         int textureSlot = 0;
         addDesiredStateChange(new SetInputTexture(
-                textureSlot++, READ_ONLY_GBUFFER.getFbo().colorBufferTextureId,   DEFERRED_LIGHTING_MATERIAL, "texSceneOpaque"));
+                textureSlot++, sceneOpaqueFbo.colorBufferTextureId,   DEFERRED_LIGHTING_MATERIAL, "texSceneOpaque"));
```

Comment:
Yeah, I saw this problem a long time ago (when I was using InitRender() hack, I compared the old and new taskGraphs and manually compared each node, and figured out that ApplyDeferredLightingNode and BindFBO were causing problems). Working on this right now :D